current_spring2026/graph_extract_spacy.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash -l
+#$ -P sparkgrp
+#$ -l h_rt=12:00:00
+#$ -N graph-metadata-extract
+#$ -j y
+#$ -o logs/graph-metadata-extract.log
+#$ -pe omp 8
+
+module load miniconda
+conda activate spark-rag
+
+cd /projectnb/sparkgrp/ml-bpl-rag-data-subset/temp/ml-bpl-rag/current_spring2026
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)